src/utils/consts.ts

```diff
@@ -479,6 +496,22 @@ export const NEAR_CORE_BRIDGE_ACCOUNT =
 export const NEAR_TOKEN_BRIDGE_ACCOUNT =
   CLUSTER === "testnet" ? "token.wormhole.testnet" : "token.test.near";
 
+export const MOONBEAM_BRIDGE_ADDRESS = getAddress(
```

Comment:
Can we remove these? They appear to be unused

Reply:
Done